atlas/getting-started/repository-settings.mdx

```diff
@@ -44,17 +42,17 @@ To use the Faust.js starter kit, do the following:
 
 ![Create a new repo](./assets/img/new-repo-screen.png)
 
-4. Navigate back to the WP Engine User Portal. Select your new repository from the repository list.
+4. Navigate back to the WP Engine User Portal by closing the GitHub tab. Select your new repository from the repository list when it refreshes.
```

Comment:
This is a good clarification for the user, as this was a common sticking point in some of the UX interviews on the blueprint flow.